### DISCUSSION TITLE
How do i make Github site subdomain

### DISCUSSION THREAD

**Discussion by Viksevik** [community/community#56199] [New to GitHub] [answered]
I have tried following the documentation on github, and also tried to follow a youtube video. But i don't know how i make a github site in this format x.github.io where the x would be something i choose. When i do what the documentation and youtube video say it turns out like this https://viksevik.github.io/omniunit.github.io/ (the current site is also just a test). Does anyone know what i might be doing wrong?

**Comment by airtower-luna** [comment, marked as answer]
> But i don't know how i make a github site in this format x.github.io where the x would be something i choose.

The "x" there *must* be the name of the user or organization owning the repository, that's the only `github.io` domain you get. You'll have to either use `viksevik.github.io` or see if you can create an organization with the name you want (and have it own the repository).
